### PR TITLE
fofb-epics-ioc-slot-mapping: add missing and standardize dev crates

### PR DESCRIPTION
**Pull request by guilhermerc**
Development crates prefixes start with 'DE'and end with 'SLxx'. The map of {crate number, crate} is: {22, Homolog}, {23, Tests} and {24, Minicrate}.

### CODE REVIEW

File: scripts/systemd/etc/sysconfig/fofb-epics-ioc-slot-mapping [hunk 1]
```diff
@@ -2077,300 +2077,249 @@ CRATE_21_FOFB_23_PV_DEVICE_PREFIX=RaBPM:BS-FOFBCtrl:
 CRATE_21_FOFB_24_PV_AREA_PREFIX=XX-21SL24
 CRATE_21_FOFB_24_PV_DEVICE_PREFIX=RaBPM:BS-FOFBCtrl:
 
+# --- CRATE 22 (Homolog) ---
 
-# --- CRATE 22 ---
+# Crate 22 - FOFB slot 1 - DE-22SL01:RaBPM:BS-FOFBCtrl
```
Comment (ericonr) [inline scripts/systemd/etc/sysconfig/fofb-epics-ioc-slot-mapping:2082]: I was thinking `DE-22SL01:BS-FOFBCtrl`. Do you think we need the `RaBPM`part?

File: scripts/systemd/etc/sysconfig/fofb-epics-ioc-slot-mapping [hunk 2]
```diff
@@ -2077,300 +2077,249 @@ CRATE_21_FOFB_23_PV_DEVICE_PREFIX=RaBPM:BS-FOFBCtrl:
...
+CRATE_22_BPM_1_PV_AREA_PREFIX=DE-22SL01:
+CRATE_22_BPM_1_PV_DEVICE_PREFIX=RaBPM:BS-FOFBCtrl:
 
-# Crate 22 - FOFB slot 1 - XX-22SL01RaBPM:BS-FOFBCtrl
-CRATE_22_FOFB_1_PV_AREA_PREFIX=XX-22SL01
-CRATE_22_FOFB_1_PV_DEVICE_PREFIX=RaBPM:BS-FOFBCtrl:
+# Crate 22 - FOFB slot 2 - DE-22SL02:RaBPM:BS-FOFBCtrl
+CRATE_22_BPM_2_PV_AREA_PREFIX=DE-22SL02:
```
Comment (ericonr) [inline scripts/systemd/etc/sysconfig/fofb-epics-ioc-slot-mapping:2087]: Changed to `CRATE_22_BPM_2_PV`; the correct is `CRATE_22_FOFB_2_PV`

File: scripts/systemd/etc/sysconfig/fofb-epics-ioc-slot-mapping (hunk 1 above)
Comment (guilhermerc) [reply scripts/systemd/etc/sysconfig/fofb-epics-ioc-slot-mapping:2082]: No, I don't, just kept it bc all the other macros have...

Comment (guilhermerc) [reply scripts/systemd/etc/sysconfig/fofb-epics-ioc-slot-mapping:2082]: I suggest to append `RaBPM` to all `*FOFB_{3,5}_PV_AREA_PREFIX*` and remove it from all `*PV_DEVICE_PREFIX`. By doing that we don't break any PV name being used and also keep consistency with [tim-rx-epics-ioc-slot-mapping](https://github.com/lnls-dig/tim-rx-epics-ioc/blob/fed3ea3bdb13bc083d79e0fc1884dc55546a6c7c/scripts/systemd/etc/sysconfig/tim-rx-epics-ioc-slot-mapping). What do you think?

Comment (guilhermerc) [reply scripts/systemd/etc/sysconfig/fofb-epics-ioc-slot-mapping]: Addressed on 2dd6dc49f41c519e628355aec210a5f7c2483181.

File: scripts/systemd/etc/sysconfig/fofb-epics-ioc-slot-mapping [hunk 3]
```diff
@@ -3,2374 +3,2323 @@
...
 
 # Crate 03 - FOFB slot 2 - XX-03SL02RaBPM:BS-FOFBCtrl
 CRATE_03_FOFB_2_PV_AREA_PREFIX=XX-03SL02
-CRATE_03_FOFB_2_PV_DEVICE_PREFIX=RaBPM:BS-FOFBCtrl:
+CRATE_03_FOFB_2_PV_DEVICE_PREFIX=BS-FOFBCtrl:
 
 # Crate 03 - FOFB slot 3 - IA-03RaBPM:BS-FOFBCtrl
-CRATE_03_FOFB_3_PV_AREA_PREFIX=IA-03
-CRATE_03_FOFB_3_PV_DEVICE_PREFIX=RaBPM:BS-FOFBCtrl:
+CRATE_03_FOFB_3_PV_AREA_PREFIX=IA-03RaBPM
```
Comment (ericonr) [inline scripts/systemd/etc/sysconfig/fofb-epics-ioc-slot-mapping:211]: Starting with this one, they are missing the colon in the end. "03RaBPM:"

File: scripts/systemd/etc/sysconfig/fofb-epics-ioc-slot-mapping [hunk 4]
```diff
@@ -3,2374 +3,2323 @@
 
 # Crate 01 - FOFB slot 1 - XX-01SL01RaBPM:BS-FOFBCtrl
 CRATE_01_FOFB_1_PV_AREA_PREFIX=XX-01SL01
```
Comment (ericonr) [inline scripts/systemd/etc/sysconfig/fofb-epics-ioc-slot-mapping:5]: I feel like all of these need a colon in the end as well. "01SL01:BS-FOFBCtrl:"

File: scripts/systemd/etc/sysconfig/fofb-epics-ioc-slot-mapping [hunk 5]
```diff
@@ -1,2376 +1,2328 @@
...
-CRATE_03_FOFB_2_PV_AREA_PREFIX=XX-03SL02
-CRATE_03_FOFB_2_PV_DEVICE_PREFIX=RaBPM:BS-FOFBCtrl:
+# Crate 03 - FOFB slot 2 - XX-03SL02:BS-FOFBCtrl
+CRATE_03_FOFB_2_PV_AREA_PREFIX=XX-03SL02:
+CRATE_03_FOFB_2_PV_DEVICE_PREFIX=BS-FOFBCtrl:
 
 # Crate 03 - FOFB slot 3 - IA-03RaBPM:BS-FOFBCtrl
-CRATE_03_FOFB_3_PV_AREA_PREFIX=IA-03
-CRATE_03_FOFB_3_PV_DEVICE_PREFIX=RaBPM:BS-FOFBCtrl:
+CRATE_03_FOFB_3_PV_AREA_PREFIX=IA-03RaBPM::
```
Comment (ericonr) [inline scripts/systemd/etc/sysconfig/fofb-epics-ioc-slot-mapping:211]: Double colon

File: scripts/systemd/etc/sysconfig/fofb-epics-ioc-slot-mapping [hunk 6]
```diff
@@ -1,2376 +1,2328 @@
...
-CRATE_03_FOFB_4_PV_AREA_PREFIX=XX-03SL04
-CRATE_03_FOFB_4_PV_DEVICE_PREFIX=RaBPM:BS-FOFBCtrl:
+# Crate 03 - FOFB slot 4 - XX-03SL04:BS-FOFBCtrl
+CRATE_03_FOFB_4_PV_AREA_PREFIX=XX-03SL04:
+CRATE_03_FOFB_4_PV_DEVICE_PREFIX=BS-FOFBCtrl:
 
 # Crate 03 - FOFB slot 5 - IA-03RaBPM:BS-FOFBCtrl
-CRATE_03_FOFB_5_PV_AREA_PREFIX=IA-03
-CRATE_03_FOFB_5_PV_DEVICE_PREFIX=RaBPM:BS-FOFBCtrl:
+CRATE_03_FOFB_5_PV_AREA_PREFIX=IA-03RaBPM::
```
Comment (ericonr) [inline scripts/systemd/etc/sysconfig/fofb-epics-ioc-slot-mapping:219]: Here as well, and for the ones below it.

**Comment by ericonr** [comment]
Hmm, this part will still be wrong: https://github.com/lnls-dig/fofb-epics-ioc/search?q=EPICS_RTM_SECTION_PREFIX

**Pull request merged**